examples/tutorial/hello_world/hello_world.py

```diff
@@ -18,8 +18,8 @@ def main():
     view = Main(message="Hello World, from Python!")
     view.show()
 
-    # Start the application event loop
-    app.start()
+    # # Start the application event loop
+    # app.start()
```

Comment:
?

Reply:
Need to be removed. Helped me debug the qt5 issue on Linux.